.github/workflows/clt_nightly.yml

```diff
@@ -20,6 +20,8 @@ on:
       - 'test/clt-tests/indexer-latest-mysql-postgres/**'
       - 'test/clt-tests/core/test-alter-rename-nightly'
       - 'test/clt-tests/integrations/**'
+      - 'test/clt-tests/core/test-performance-for-queries-with-multiple-disk-chunks'
+      - 'test/clt-tests/optimisation-and-update/**'
```

Comment:
If we test it per-commit, why do we also test it nightly? Do the tests differ?

Reply:
Performance tests take longer, so run in nightly tests. Duplication is erroneous and will be removed.